lib/phlex/sgml.rb

```diff
@@ -206,6 +206,38 @@ def capture(*args, &block)
 			end
 		end
 
+		def tag(name, ...)
+			normalized_name = case name
+				when Symbol then name.name.downcase
+				when String then name.downcase
+				else raise ArgumentError.new("Expected the tag namea as a Symbol or String.")
+			end
+
+			if name == "script"
```

Comment:
```suggestion
			if normalized_name == "script"
```
Otherwise you can slip this check with "SCRIPT"

Reply:
Ops! I renamed `name` to `normalized_name` and missed this one.